content/articles/mern-isp-management-system/index.md

```diff
@@ -0,0 +1,611 @@
+                    <h2 style={headerStyle}>Update Customer Details</h2>
+                    <Typography variant='caption' gutterBottom>Edit the details below.</Typography>
+                </Grid>
+                <form>
+                    <TextField style={marginTop} id="name" value={customer.name} onChange={(e) => showCustomer({ ...customer, name: e.target.value })}
+                        fullWidth label='' placeholder="Enter your name" />
+                    <TextField style={marginTop} id="email" value={customer.email} onChange={(e) => showCustomer({ ...customer, email: e.target.value })}
+                        fullWidth label='' placeholder="Enter your email" />
+                    <TextField style={marginTop} id="phoneNumber" value={customer.phoneNumber} onChange={(e) => showCustomer({ ...customer, phoneNumber: e.target.value })}
+                        fullWidth label='' placeholder="Enter your phone number" />
+                    <TextField style={marginTop} id="location" value={customer.location} onChange={(e) => showCustomer({ ...customer, location: e.target.value })}
+                        fullWidth label='' />
+                    <TextField style={marginTop} id="location" value={customer.subscribedToPackage} onChange={(e) => showCustomer({ ...customer, subscribedToPackage: e.target.value })}
+                        fullWidth label='' />
+        
+                    <Button type='submit' variant='contained' color='primary' style={marginTop} onClick={updateCustomer}>Update</Button>
+                </form>
+            </Paper>
+        </Grid>
+    )
+}
+```
+The updateCustomer function updates the customer details in the database.
+Our EditCustomer component renders out as follows
+
+
+![Update Customer](/mern-isp-management-system/updateCustomer.png)
+
+
+Finally, we need to resolve the routes in our react app to ensure we render the desired page for a given path. 
+
+In App.js
+```JavaScript
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import CreateCustomer from './components/createCustomer';
+import CustomerList from './components/customerList';
+import EditCustomer from './components/editCustomer';
+
+function App() {
+    return (
+        <BrowserRouter>
+            <Routes>
+                <Route path="/customer" element={<CreateCustomer />} />
+                <Route path='/customerlist' element={<CustomerList />} />
+                <Route path='/edit/:id' element={<EditCustomer />} />
+                <Route exact path='/' element={<CreateCustomer />} />
+
+            </Routes>
+        </BrowserRouter>
+
+    );
+}
+
+export default App;
+```
+
+## Conclusion.<a name="chapter-3"></a>
+
+We have successfully created a management system for ISPs.
+We learned how to create REST API in Node.js, connect to the MongoDB database and create a frontend with react.
+We also learned how to connect the backend and frontend by relying on Axios and react hooks.
```

Comment:
Take a look at conclusions in other articles.

Add key takeaways, give a summar of what was covered and add reference links.

Also, add GitHub repo URL containing sourcecode